test/es-module/test-esm-wasm.mjs

```diff
@@ -1,37 +1,43 @@
-// Flags: --experimental-wasm-modules
-import '../common/index.mjs';
+import { spawnPromisified } from '../common/index.mjs';
 import { path } from '../common/fixtures.mjs';
-import { add, addImported } from '../fixtures/es-modules/simple.wasm';
-import { state } from '../fixtures/es-modules/wasm-dep.mjs';
-import { strictEqual, ok } from 'assert';
-import { spawn } from 'child_process';
+import { strictEqual, match } from 'node:assert';
+import { execPath } from 'node:process';
+import { describe, it } from 'node:test';
 
-strictEqual(state, 'WASM Start Executed');
 
-strictEqual(add(10, 20), 30);
+describe('ESM: WASM modules', { concurrency: true }, () => {
+  it('should load exports', async () => {
+    const { code, stderr, stdout } = await spawnPromisified(execPath, [
+      '--no-warnings',
+      '--experimental-wasm-modules',
+      '--input-type=module',
+      '--eval',
+      [
+        'import { strictEqual, match } from "node:assert";',
+        `import { add, addImported } from "${path('es-modules/simple.wasm')}";`,
+        `import { state } from "${path('es-modules/wasm-dep.mjs')}";`,
```

Comment:
Import specifiers are file URLs, not paths. Also, using `JSON.stringify` ensures that quotes would be correctly escaped if the file URL contains one for whatever reason.
```suggestion
        `import { add, addImported } from ${JSON.stringify(fixtures.fileURL('es-modules/simple.wasm'))};`,
        `import { state } from ${JSON.stringify(fixtures.fileURL('es-modules/wasm-dep.mjs'))};`,
```